priv/repo/seeds.exs

```diff
@@ -612,8 +676,24 @@ valtimer_params = %{
   dash_skill_id: skills["valt_warp"]
 }
 
+kenzu_params = %{
+  name: "kenzu",
+  active: Mix.env() != :prod,
```

Comment:
We might have to set this false because for some reason, it doesn't let me get into the game (from unity editor) if kenzu is active

Reply:
You will not be able to test kenzu in unity unless you test it in the respective branch which is not finished yet, this pr only adds the combo mechanic

Reply:
Done